core/controllers/base.py

```diff
@@ -156,7 +157,8 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
             user_settings = user_services.get_user_settings(
                 self.user_id, strict=False)
             if user_settings is None:
-                email = current_user_services.get_user_email(self.user)
+                email = current_user_services.get_user_email(
```

Comment:
Ditto. But I'm also wondering, do we actually need the user email (do we use it anywhere)? If not, we might want to just get rid of this part.